### PR TITLE
List the used minimum versions

### PR DESCRIPTION
**Pull request by agaida**

### CODE REVIEW

**Comment by tsujan** [comment]
Yes, it's a good idea and the list should be updated when needed.

**Comment by agaida** [comment]
right - and given that a full checkout is possible one can now grep through our whole code. That was my intention as it ease the maintenance.

File: README.md [hunk 1]
```diff
@@ -31,3 +31,18 @@ one day.)
...
 This package is arch-independent now.  You can simply package it as
 `BuildArch: noarch` (rpm) or `arch: all` (deb).
+
+## Current Minimum Versions
+
+|                              |        |
+|------------------------------|--------|
+| KF5_MINIMUM_VERSION          | 5.36.0 |
+| KF5SCREEN_MINIMUM_VERSION    | 5.2.0  |
+| LIBFM_QT_MINIMUM_VERSION     | 0.14.0 |
```
Comment (yan12125) [inline README.md]: This line can be removed after https://github.com/lxqt/lxqt-archiver/pull/54

File: README.md [hunk 2]
```diff
@@ -31,3 +31,18 @@ one day.)
 
 This package is arch-independent now.  You can simply package it as
 `BuildArch: noarch` (rpm) or `arch: all` (deb).
+
+## Current Minimum Versions
+
+|                              |        |
```
Comment (yan12125) [inline README.md]: The captions are missing. How about "Package" and "Version"?

Comment (yan12125) [reply README.md]: Done :)

**Pull request merged**